### ISSUE TITLE
AIP 2: Airdrop for Holders

### ISSUE THREAD

**Issue by dmosites** [airswap/airswap-aips#2]
# Summary
On October 10th, 2017 the AirSwap token launched to 9,447 people from 135 countries. While both AirSwap and the ecosystem have changed since then, many token holders have continuously held and used their AirSwap tokens as the network and community have evolved.

Today, the token plays a role in governance, and we believe that it’s in the best interest of the project to empower longtime holders with an even greater voice in the governance process. We propose to award community members with additional AST based on how long they’ve held.

# Specification
Once accepted, a date will be determined to snapshot Ethereum accounts that hold a balance of AirSwap tokens at that time. For each account, a script will determine for how long it has held a nonzero balance of AirSwap tokens and airdrop tokens pro rata to each account.

**Voting**. The vote will be to execute a trial airdrop of 100K AST to holders based on the amount of time they’ve held their tokens. Vote on this proposal at [https://vote.airswap.io/](https://vote.airswap.io/#/airswap/proposal/QmNZDMgeuaHXBkqo7ywggKwVEe5Lg4Fet6Y1WTi9fsAZsH)

_For more information on the AIP process see AIP #1._

# Rationale
This proposal aims to bootstrap the governance process and positively impact the community, especially as combined with other AIPs. As for calculation, it’s arguably in the best interest of the token community to reward members based on the amount of time held rather than the number of tokens held.

# Copyright
Copyright and related rights waived via CC0.

**Comment by Synpaw** [comment]
All the old holders hold on for so long only because they have a small amount of AST, and were not ready to sell at a bad price
I believe that we need to find a different approach to distribution
Or not to spend it at all

**Comment by LupusDei** [comment]
Agreed, an airdrop, or series of airdrops, do not provide incentives to become active in the protocol/ecosystem in the same way that rewarding LPs do.  The tend to just encourage passive holding and accumulation.

**Comment by Tzouf** [comment]
What about AST on exchanges?

**Comment by mtrigga** [comment]
Can these airdropped tokens be time-locked? This would increase the likelihood the tokens are used by long-term stakeholders in the ecosystem and will not be dumped on the market.

**Comment by fawazahmed** [comment]
An Airdrop wouldn't necessarily have an impact on Governance. But, it would cause Sell Pressure and lots of confusion. Issues such as multiple wallets, people who've gotten hacked, what platform people bought/held $AST on will be very tricky to navigate.

If anyone really wants to have an impact on Governance, they can always buy more $AST, or earn it with their impact on Governance: if we could figure out a way to reward Active Community Members (active on Forums, Discord, AIPs), that's probably the best way of helping increase their voting power and, therefore, impact. Our Community could vote on who to award more tokens to in the future depending on the impact they have on Airswap's direction with their AIPs. Rewarding people who put their time and effort into helping steer Airswap's direction is better than Airdropping to Wallets that might never vote, but rather just sell....

**Comment by Nobs0x** [comment]
I am heavily against doing an airdrop. It will result in selling many tokens, which brings confusion and will push away new people from the project.
 
Instead of doing an airdrop, most of the tokens should be burned. Some may be allocated in a treasury, which will give additional security and bring confidence to the newcomers and the people who are here for the long term period.
If any "free" token distribution is made, it should be after a certain period of time, where all LP's can claim some "free" tokens (fixed and equal amount for every LP). This will incentivize people to hold and not sell. However, I am not advocating for it. I think this is just the best option when we talk about airdropping.

I think burning the tokens will be better due to the reason that the price will increase and can stabilize on higher levels. This will not create massive sell out due to the higher price, and a smaller number of the tokens will be sold. 
Having **incentives**, like liquidity rewards and fee-sharing, will make new people buy the tokens sold, and these people will be here for these incentives. It's a win-win move.

**Comment by muratTokensfund** [comment]
how do I vote against airdrop?

**Comment by Nobs0x** [comment]
@muratTokensfund , The proposal is now in its "Draft" stage and is not submitted as an official proposal. Once it is, we can vote. However, I sincerely hope that this proposal will not be submitted for voting. Like you, I am against it.

**Comment by ejwessel** [comment]
https://vote.airswap.io/#/airswap/proposal/QmNZDMgeuaHXBkqo7ywggKwVEe5Lg4Fet6Y1WTi9fsAZsH
